### ISSUE TITLE
Error in upgrade from 4.8.0 to 4.8.1

### ISSUE THREAD

**Issue by rafabailon** [wazuh/wazuh#24489]
| Wazuh Version | Component | Install Type | Install Method | Platform |
| --- | --- | --- | --- | --- |
| 4.8.1-rc2 | Wazuh Manager | Manager | Packages | Different Operating Systems |

During [Release 4.8.1 - RC 2 - Upgrade metrics](https://github.com/wazuh/wazuh/issues/24430) testing, errors have been detected in the Manager for different Operating Systems. The error occurs when trying to upgrade the Manager from version 4.8.0 to version 4.8.1.

<details>
   <summary>Logs</summary>
   <br/>

```console
19:03:49  TASK [test-manager : Upgrade test manager (Pytest)] ****************************
19:03:49  task path: /home/ec2-user/workspace/Test_upgrade@2/quality/tests/generic/03-upgrade/linux/test/conf/test-manager/tasks/configure_test.yaml:51
19:03:49  fatal: [amd64-fedora-36-manager-4_8_0_0]: FAILED! => {
19:03:49      "changed": true,
19:03:49      "cmd": [
19:03:49          "/usr/testing/bin/pytest",
19:03:49          "--tb=short",
19:03:49          "upgrade_manager_test.py",
19:03:49          "--from_version",
19:03:49          "4.8.0",
19:03:49          "--to_version",
19:03:49          "4.8.1",
19:03:49          "--os",
19:03:49          "fedora 36",
19:03:49          "-v"
19:03:49      ],
19:03:49      "delta": "0:00:00.521066",
19:03:49      "end": "2024-07-05 17:03:49.688508",
19:03:49      "rc": 1,
19:03:49      "start": "2024-07-05 17:03:49.167442"
19:03:49  }
19:03:49  
19:03:49  STDOUT:
19:03:49  
19:03:49  ============================= test session starts ==============================
19:03:49  platform linux -- Python 3.8.2, pytest-5.4.1, py-1.11.0, pluggy-0.13.1 -- /usr/testing/bin/python3.8
19:03:49  cachedir: .pytest_cache
19:03:49  rootdir: /upgrade_manager
19:03:49  collecting ... collected 19 items
19:03:49  
19:03:49  upgrade_manager_test.py::test_check_version PASSED                       [  5%]
19:03:49  upgrade_manager_test.py::test_check_process_clusterd PASSED              [ 10%]
19:03:49  upgrade_manager_test.py::test_check_process_modulesd PASSED              [ 15%]
19:03:49  upgrade_manager_test.py::test_check_process_monitord PASSED              [ 21%]
19:03:49  upgrade_manager_test.py::test_check_process_logcollector PASSED          [ 26%]
19:03:49  upgrade_manager_test.py::test_check_process_remoted PASSED               [ 31%]
19:03:49  upgrade_manager_test.py::test_check_process_syscheckd PASSED             [ 36%]
19:03:49  upgrade_manager_test.py::test_check_process_analysisd PASSED             [ 42%]
19:03:49  upgrade_manager_test.py::test_check_process_maild PASSED                 [ 47%]
19:03:49  upgrade_manager_test.py::test_check_process_execd PASSED                 [ 52%]
19:03:49  upgrade_manager_test.py::test_check_process_wazuhdb PASSED               [ 57%]
19:03:49  upgrade_manager_test.py::test_check_log_errors FAILED                    [ 63%]
19:03:49  upgrade_manager_test.py::test_check_system_ossec_conf PASSED             [ 68%]
19:03:49  upgrade_manager_test.py::test_check_ossec_conf PASSED                    [ 73%]
19:03:49  upgrade_manager_test.py::test_check_client_keys_conf PASSED              [ 78%]
19:03:49  upgrade_manager_test.py::test_check_local_internal_options_conf PASSED   [ 84%]
19:03:49  upgrade_manager_test.py::test_check_local_decoder_conf PASSED            [ 89%]
19:03:49  upgrade_manager_test.py::test_check_local_rules_conf PASSED              [ 94%]
19:03:49  upgrade_manager_test.py::test_check_agent_conf PASSED                    [100%]
19:03:49  
19:03:49  =================================== FAILURES ===================================
19:03:49  ____________________________ test_check_log_errors _____________________________
19:03:49  upgrade_manager_test.py:178: in test_check_log_errors
19:03:49      assert expected_error_found == True, assert_description['error']['exception_not_detected']
19:03:49  E   AssertionError: An error was found in the log
19:03:49  E   assert False == True
19:03:49  E     +False
19:03:49  E     -True
19:03:49  ----------------------------- Captured stdout call -----------------------------
19:03:49  2024/07/05 16:59:55 wazuh-modulesd:vulnerability-scanner: INFO: Database decompression finished.
19:03:49  
19:03:49  2024/07/05 16:59:56 wazuh-modulesd:vulnerability-scanner: INFO: Vulnerability scanner module started
19:03:49  
19:03:49  2024/07/05 16:59:56 indexer-connector: WARNING: Failed to sync agent '001' with the indexer.
19:03:49  
19:03:49  2024/07/05 17:00:33 wazuh-modulesd:syscollector: INFO: Stop received for Syscollector.
19:03:49  
19:03:49  2024/07/05 17:00:33 wazuh-modulesd:syscollector: INFO: Module finished.
19:03:49  
19:03:49  2024/07/05 17:00:33 wazuh-modulesd:vulnerability-scanner: INFO: Stopping vulnerability_scanner module.
19:03:49  
19:03:49  2024/07/05 17:00:33 wazuh-modulesd:router: INFO: Stopping router module.
19:03:49  
19:03:49  2024/07/05 17:00:33 wazuh-modulesd:content_manager: INFO: Stopping content_manager module.
19:03:49  
19:03:49  2024/07/05 17:00:33 wazuh-modulesd:content-updater: WARNING: The offsets download has been interrupted
19:03:49  
19:03:49  2024/07/05 17:00:33 wazuh-monitord: INFO: (1225): SIGNAL [(15)-(Terminated)] Received. Exit Cleaning...
19:03:49  
19:03:49  2024/07/05 17:00:33 wazuh-logcollector: INFO: (1225): SIGNAL [(15)-(Terminated)] Received. Exit Cleaning...
19:03:49  
19:03:49  2024/07/05 17:00:33 wazuh-remoted: INFO: (1225): SIGNAL [(15)-(Terminated)] Received. Exit Cleaning...
19:03:49  
19:03:49  2024/07/05 17:00:33 wazuh-syscheckd: INFO: (1756): Shutdown received. Releasing resources.
19:03:49  
19:03:49  2024/07/05 17:00:33 wazuh-syscheckd: INFO: (1225): SIGNAL [(15)-(Terminated)] Received. Exit Cleaning...
19:03:49  
19:03:49  2024/07/05 17:00:33 wazuh-analysisd: INFO: (1225): SIGNAL [(15)-(Terminated)] Received. Exit Cleaning...
19:03:49  
19:03:49  2024/07/05 17:00:33 wazuh-execd: INFO: (1314): Shutdown received. Deleting responses.
19:03:49  
19:03:49  2024/07/05 17:00:33 wazuh-execd: INFO: (1225): SIGNAL [(15)-(Terminated)] Received. Exit Cleaning...
19:03:49  
19:03:49  2024/07/05 17:00:33 wazuh-db: INFO: (1225): SIGNAL [(15)-(Terminated)] Received. Exit Cleaning...
19:03:49  
19:03:49  2024/07/05 17:00:34 wazuh-db: INFO: Graceful process shutdown.
19:03:49  
19:03:49  2024/07/05 17:00:34 wazuh-authd: INFO: (1225): SIGNAL [(15)-(Terminated)] Received. Exit Cleaning...
19:03:49  
19:03:49  2024/07/05 17:00:35 wazuh-authd: INFO: Exiting...
19:03:49  
19:03:49  2024/07/05 17:00:58 wazuh-modulesd:router: INFO: Loaded router module.
19:03:49  
19:03:49  2024/07/05 17:00:58 wazuh-modulesd:content_manager: INFO: Loaded content_manager module.
19:03:49  
19:03:49  2024/07/05 17:01:00 wazuh-csyslogd: INFO: Remote syslog server not configured. Clean exit.
19:03:49  
19:03:49  2024/07/05 17:01:00 wazuh-dbd: INFO: Database not configured. Clean exit.
19:03:49  
19:03:49  2024/07/05 17:01:00 wazuh-integratord: INFO: Remote integrations not configured. Clean exit.
19:03:49  
19:03:49  2024/07/05 17:01:00 wazuh-agentlessd: INFO: Not configured. Exiting.
19:03:49  
19:03:49  2024/07/05 17:01:01 wazuh-authd: INFO: Started (pid: 2790).
19:03:49  
19:03:49  2024/07/05 17:01:01 wazuh-authd: INFO: Accepting connections on port 1515. No password required.
19:03:49  
19:03:49  2024/07/05 17:01:01 wazuh-authd: INFO: Setting network timeout to 1.000000 sec.
19:03:49  
19:03:49  2024/07/05 17:01:02 wazuh-db: INFO: Started (pid: 2806).
19:03:49  
19:03:49  2024/07/05 17:01:03 wazuh-execd: INFO: Started (pid: 2833).
19:03:49  
19:03:49  2024/07/05 17:01:05 wazuh-syscheckd: INFO: Started (pid: 2861).
19:03:49  
19:03:49  2024/07/05 17:01:05 wazuh-syscheckd: INFO: (6003): Monitoring path: '/bin', with options 'size | permissions | owner | group | mtime | inode | hash_md5 | hash_sha1 | hash_sha256 | scheduled'.
19:03:49  
19:03:49  2024/07/05 17:01:05 wazuh-syscheckd: INFO: (6003): Monitoring path: '/boot', with options 'size | permissions | owner | group | mtime | inode | hash_md5 | hash_sha1 | hash_sha256 | scheduled'.
19:03:49  
19:03:49  2024/07/05 17:01:05 wazuh-syscheckd: INFO: (6003): Monitoring path: '/etc', with options 'size | permissions | owner | group | mtime | inode | hash_md5 | hash_sha1 | hash_sha256 | scheduled'.
19:03:49  
19:03:49  2024/07/05 17:01:05 wazuh-syscheckd: INFO: (6003): Monitoring path: '/sbin', with options 'size | permissions | owner | group | mtime | inode | hash_md5 | hash_sha1 | hash_sha256 | scheduled'.
19:03:49  
19:03:49  2024/07/05 17:01:05 wazuh-syscheckd: INFO: (6003): Monitoring path: '/usr/bin', with options 'size | permissions | owner | group | mtime | inode | hash_md5 | hash_sha1 | hash_sha256 | scheduled'.
19:03:49  
19:03:49  2024/07/05 17:01:05 wazuh-syscheckd: INFO: (6003): Monitoring path: '/usr/sbin', with options 'size | permissions | owner | group | mtime | inode | hash_md5 | hash_sha1 | hash_sha256 | scheduled'.
19:03:49  
19:03:49  2024/07/05 17:01:05 wazuh-syscheckd: INFO: (6206): Ignore 'file' entry '/etc/mtab'
19:03:49  
19:03:49  2024/07/05 17:01:05 wazuh-syscheckd: INFO: (6206): Ignore 'file' entry '/etc/hosts.deny'
19:03:49  
19:03:49  2024/07/05 17:01:05 wazuh-syscheckd: INFO: (6206): Ignore 'file' entry '/etc/mail/statistics'
19:03:49  
19:03:49  2024/07/05 17:01:05 wazuh-syscheckd: INFO: (6206): Ignore 'file' entry '/etc/random-seed'
19:03:49  
19:03:49  2024/07/05 17:01:05 wazuh-syscheckd: INFO: (6206): Ignore 'file' entry '/etc/random.seed'
19:03:49  
19:03:49  2024/07/05 17:01:05 wazuh-syscheckd: INFO: (6206): Ignore 'file' entry '/etc/adjtime'
19:03:49  
19:03:49  2024/07/05 17:01:05 wazuh-syscheckd: INFO: (6206): Ignore 'file' entry '/etc/httpd/logs'
19:03:49  
19:03:49  2024/07/05 17:01:05 wazuh-syscheckd: INFO: (6206): Ignore 'file' entry '/etc/utmpx'
19:03:49  
19:03:49  2024/07/05 17:01:05 wazuh-syscheckd: INFO: (6206): Ignore 'file' entry '/etc/wtmpx'
19:03:49  
19:03:49  2024/07/05 17:01:05 wazuh-syscheckd: INFO: (6206): Ignore 'file' entry '/etc/cups/certs'
19:03:49  
19:03:49  2024/07/05 17:01:05 wazuh-syscheckd: INFO: (6206): Ignore 'file' entry '/etc/dumpdates'
19:03:49  
19:03:49  2024/07/05 17:01:05 wazuh-syscheckd: INFO: (6206): Ignore 'file' entry '/etc/svc/volatile'
19:03:49  
19:03:49  2024/07/05 17:01:05 wazuh-syscheckd: INFO: (6207): Ignore 'file' sregex '.log$|.swp$'
19:03:49  
19:03:49  2024/07/05 17:01:05 wazuh-syscheckd: INFO: (6004): No diff for file: '/etc/ssl/private.key'
19:03:49  
19:03:49  2024/07/05 17:01:05 wazuh-syscheckd: INFO: (6000): Starting daemon...
19:03:49  
19:03:49  2024/07/05 17:01:05 wazuh-syscheckd: INFO: (6010): File integrity monitoring scan frequency: 43200 seconds
19:03:49  
19:03:49  2024/07/05 17:01:05 wazuh-syscheckd: INFO: (6008): File integrity monitoring scan started.
19:03:49  
19:03:49  2024/07/05 17:01:05 rootcheck: INFO: Starting rootcheck scan.
19:03:49  
19:03:49  2024/07/05 17:01:06 wazuh-analysisd: INFO: Total rules enabled: '6786'
19:03:49  
19:03:49  2024/07/05 17:01:06 wazuh-analysisd: INFO: Started (pid: 2847).
19:03:49  
19:03:49  2024/07/05 17:01:06 wazuh-analysisd: INFO: EPS limit disabled
19:03:49  
19:03:49  2024/07/05 17:01:06 wazuh-analysisd: INFO: (7200): Logtest started
19:03:49  
19:03:49  2024/07/05 17:01:06 wazuh-remoted: INFO: Started (pid: 2908). Listening on port 1514/TCP (secure).
19:03:49  
19:03:49  2024/07/05 17:01:06 wazuh-remoted: INFO: (1410): Reading authentication keys file.
19:03:49  
19:03:49  2024/07/05 17:01:07 wazuh-logcollector: INFO: Monitoring output of command(360): df -P
19:03:49  
19:03:49  2024/07/05 17:01:07 wazuh-logcollector: INFO: Monitoring full output of command(360): netstat -tulpn | sed 's/\([[:alnum:]]\+\)\ \+[[:digit:]]\+\ \+[[:digit:]]\+\ \+\(.*\):\([[:digit:]]*\)\ \+\([0-9\.\:\*]\+\).\+\ \([[:digit:]]*\/[[:alnum:]\-]*\).*/\1 \2 == \3 == \4 \5/' | sort -k 4 -g | sed 's/ == \(.*\) ==/:\1/' | sed 1,2d
19:03:49  
19:03:49  2024/07/05 17:01:07 wazuh-logcollector: INFO: Monitoring full output of command(360): last -n 20
19:03:49  
19:03:49  2024/07/05 17:01:07 wazuh-logcollector: INFO: (1950): Analyzing file: '/var/ossec/logs/active-responses.log'.
19:03:49  
19:03:49  2024/07/05 17:01:07 wazuh-logcollector: INFO: Started (pid: 2949).
19:03:49  
19:03:49  2024/07/05 17:01:08 wazuh-monitord: INFO: Started (pid: 2968).
19:03:49  
19:03:49  2024/07/05 17:01:09 wazuh-modulesd:router: INFO: Loaded router module.
19:03:49  
19:03:49  2024/07/05 17:01:09 wazuh-modulesd:content_manager: INFO: Loaded content_manager module.
19:03:49  
19:03:49  2024/07/05 17:01:09 wazuh-modulesd: INFO: Started (pid: 2988).
19:03:49  
19:03:49  2024/07/05 17:01:09 wazuh-modulesd:osquery: INFO: Module disabled. Exiting...
19:03:49  
19:03:49  2024/07/05 17:01:09 sca: INFO: Module started.
19:03:49  
19:03:49  2024/07/05 17:01:09 sca: INFO: No policies defined. Exiting.
19:03:49  
19:03:49  2024/07/05 17:01:09 wazuh-modulesd:router: INFO: Starting router module.
19:03:49  
19:03:49  2024/07/05 17:01:09 wazuh-modulesd:vulnerability-scanner: INFO: Starting vulnerability_scanner module.
19:03:49  
19:03:49  2024/07/05 17:01:09 wazuh-modulesd:ciscat: INFO: Module disabled. Exiting...
19:03:49  
19:03:49  2024/07/05 17:01:09 wazuh-modulesd:agent-upgrade: INFO: (8153): Module Agent Upgrade started.
19:03:49  
19:03:49  2024/07/05 17:01:09 wazuh-modulesd:content_manager: INFO: Starting content_manager module.
19:03:49  
19:03:49  2024/07/05 17:01:09 wazuh-modulesd:task-manager: INFO: (8200): Module Task Manager started.
19:03:49  
19:03:49  2024/07/05 17:01:09 wazuh-modulesd:database: INFO: Module started.
19:03:49  
19:03:49  2024/07/05 17:01:09 wazuh-modulesd:download: INFO: Module started.
19:03:49  
19:03:49  2024/07/05 17:01:09 wazuh-modulesd:control: INFO: Starting control thread.
19:03:49  
19:03:49  2024/07/05 17:01:09 wazuh-modulesd:syscollector: INFO: Module started.
19:03:49  
19:03:49  2024/07/05 17:01:09 wazuh-modulesd:syscollector: INFO: Starting evaluation.
19:03:49  
19:03:49  2024/07/05 17:01:10 indexer-connector: WARNING: No username and password found in the keystore, using default values.
19:03:49  
19:03:49  2024/07/05 17:01:10 wazuh-syscheckd: INFO: (6009): File integrity monitoring scan ended.
19:03:49  
19:03:49  2024/07/05 17:01:10 wazuh-syscheckd: INFO: FIM sync module started.
19:03:49  
19:03:49  2024/07/05 17:01:10 indexer-connector: WARNING: IndexerConnector initialization failed for index 'wazuh-states-vulnerabilities-ip-172-31-94-15.ec2.internal', retrying until the connection is successful.
19:03:49  
19:03:49  2024/07/05 17:01:10 wazuh-modulesd:vulnerability-scanner: INFO: Starting database file decompression.
19:03:49  
19:03:49  2024/07/05 17:01:10 wazuh-modulesd:syscollector: INFO: Evaluation finished.
19:03:49  
19:03:49  2024/07/05 17:01:30 rootcheck: INFO: Ending rootcheck scan.
19:03:49  
19:03:49  2024/07/05 17:01:34 wazuh-modulesd:router: INFO: Loaded router module.
19:03:49  
19:03:49  2024/07/05 17:01:34 wazuh-modulesd:content_manager: INFO: Loaded content_manager module.
19:03:49  
19:03:49  2024/07/05 17:01:37 wazuh-csyslogd: INFO: Remote syslog server not configured. Clean exit.
19:03:49  
19:03:49  2024/07/05 17:01:38 wazuh-dbd: INFO: Database not configured. Clean exit.
19:03:49  
19:03:49  2024/07/05 17:01:38 wazuh-integratord: INFO: Remote integrations not configured. Clean exit.
19:03:49  
19:03:49  2024/07/05 17:01:38 wazuh-agentlessd: INFO: Not configured. Exiting.
19:03:49  
19:03:49  2024/07/05 17:01:53 wazuh-modulesd:vulnerability-scanner: ERROR: VulnerabilityScannerFacade::start: Couldn't open file
19:03:49  
19:03:49  =============================== warnings summary ===============================
19:03:49  upgrade_manager_test.py:135
19:03:49    /upgrade_manager/upgrade_manager_test.py:135: DeprecationWarning: invalid escape sequence \w
19:03:49      daemonRegex = '\w+-\w+:'
19:03:49  
19:03:49  -- Docs: [https://docs.pytest.org/en/latest/warnings.html](https://docs.pytest.org/en/latest/warnings.html%1B[0m)
19:03:49  =========================== short test summary info ============================
19:03:49  FAILED upgrade_manager_test.py::test_check_log_errors - AssertionError: An er...
19:03:49  =================== 1 failed, 18 passed, 1 warning in 0.11s ====================
19:03:49  
19:03:49  
19:03:49  MSG:
19:03:49  
19:03:49  non-zero return code
```

</details>

Errors occur for the following Operating Systems: CentOS 8, CentOS 7, Debian 12 (bookworm), Debian 11 (bullseye), Debian 10 (buster), Debian 9 (strech), Fedora 36, Fedora 35, Fedora 34, Fedora 32, Fedora 31, Ubuntu Focal, Ubuntu Bionic, Ubuntu Xenial, Ubuntu Trusty, Amazon Linux 2, Amazon Linux 1, Oracle Linux 8, openSUSE Tumbleweed, Red Hat 7, Red Hat 8, Red Hat 9.

For some operating systems, the test has not failed: CentOS 6, CentOS 5, Red Hat 6.

- [CentOS 6 Pipeline (Success)](https://ci.wazuh.info/job/Test_upgrade/135830/console)
- [CentOS 7 Pipeline (Failed)](https://ci.wazuh.info/job/Test_upgrade/135829/console)

It is necessary to investigate what is the cause of the errors.

**Comment by rauldpm** [comment]
Test adapted in 4.9.0 in https://github.com/wazuh/wazuh-jenkins/pull/6687
Added log improvement in 4.9.0 https://github.com/wazuh/wazuh/pull/24345

**Comment by juliamagan** [comment]
This issue is solved in 4.9.0 and the impact is very low in 4.8.1.
